### ISSUE TITLE
Which aes key

### ISSUE THREAD

**Issue by devvyyxyz** [Cracko298/UE4-AES-Key-Extracting-Guide#6]
When using the program to get an AES key how do I know which one to use?

**Comment by Cracko298** [comment]
In UEViewer you can input multiple, same with Fmodel.

**Comment by Cracko298** [comment]
Traditionally, Basic keys such as:
```
0x4F004F004F004F004F004F004F004F004F004F004F004F004F004F004F004F00
```
and
```
0x6A0000000000000000000000000000006A000000000000000000000000000000
```
Aren't actual keys, instead just used by the Engine Version itself, and not the game or False-Positives.


Normal Encryption/Decryption Keys are more complex and aren't really able to be distinguished apart.
As per my suggestion in the comment above, also keep in Mind, that repeating characters evenly spaced out are extremely uncommon for AES Keys and other Encryption Methods.

If this still doesn't answer your Question, please refer to similarly solved issues: #2 
Link: https://github.com/Cracko298/UE4-AES-Key-Extracting-Guide/issues/2#issuecomment-1574969416

Two Known Fake Keys that keep showing up are as follows and should be ignored.
```
0x0000803F0000803F0000803F0000803F000048430000F0410AD7A33C0000803F
0x9A99993E0000803F0000003F0000003F04000000060000000000003F00030303
```

**Comment by Cracko298** [comment]
Closing issue, assumed to be resolved after 3 months of inactivity.
